fmriprep/utils/bids.py

```diff
@@ -145,6 +145,9 @@ def write_derivative_description(bids_dir, deriv_dir):
     if 'License' in orig_desc:
         desc['License'] = orig_desc['License']
 
+    # Add DatasetLinks
+    desc['DatasetLinks'] = {k: str(v) for k, v in config.execution.dataset_links.items()}
```

Comment:
Generally, I've avoided accessing the configuration outside the `fmriprep.workflow` module - it makes these functions easier to test and less reliant on the beast of the workflow that fmriprep is. This is probably what is causing the issue you're seeing

Making dataset_links a (optional?) parameter to this function seems like a cleaner approach

Reply:
That's a good point. I'll change the function. Thanks!